hardware/microwave/mw_source_smbv.py

```diff
@@ -92,21 +102,21 @@ def get_limits(self):
         limits = MicrowaveLimits()
         limits.supported_modes = (MicrowaveMode.CW, MicrowaveMode.SWEEP)
 
-        limits.min_power = -145
-        limits.max_power = 30
+        limits.min_power = self._limits_min_power
+        limits.max_power = self._limits_max_power
 
-        limits.min_frequency = 9e3
-        limits.max_frequency = 6e9
+        limits.min_frequency = self._limits_min_frequency
+        limits.max_frequency = self._limits_max_frequency
 
         limits.list_minstep = 0.1
-        limits.list_maxstep = 5.991e9
+        limits.list_maxstep = self._limits_max_frequency - self._limits_min_frequency
         limits.list_maxentries = 1
 
         limits.sweep_minstep = 0.1
-        limits.sweep_maxstep = 5.991e9
+        limits.sweep_maxstep = self._limits_max_frequency - self._limits_min_frequency
         limits.sweep_maxentries = 10001
 
-        if self.model != 'SMBV100A':
```

Comment:
Keep the old style here, limits should be present for every model where they can be determined.